lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -158,12 +173,25 @@ defmodule Mix.Compilers.Elixir do
 
     kind   = detect_kind(module)
     source = Path.relative_to(source, cwd)
-    files  = get_external_resources(module, cwd)
+    source_files  = get_external_resources(module, cwd)
+
+    Agent.cast pid, fn {modules, files, sources} ->
+      new_module =
+        module(
+          module: module,
+          kind: kind,
+          source: source,
+          beam: beam,
+          binary: binary
+        )
```

Comment:
@whatyouhide Any time I assign something that requires a new level of indentation I just do it on the next line no matter what.